library/rocksdbjni/README.md

```diff
@@ -9,11 +9,13 @@ used in production.
 
 ## Deploying RocksDB JNI with Bazel
 
-0. First run `cat library/rocksdbjni/VERSION` to check that the version number is correct.
+**NOTE: Only -mac distributions are currently operational.**
```

Comment:
I think we should say `**NOTE: Only Mac distributions are currently operational.**`. Saying "`-mac`" is not very intuitive.

Reply:
Additionally, I think the README can be improved. What do you think of this structure:

Reply:
# RocksDB JNI
...
## Assembling & Deploying with Bazel

**NOTE: Only Mac distributions are currently operational.**

1. Update `VERSION` file with the version of RocksDB JNI that you wish to assemble & deploy:
```
echo "<rocksdb-jni-version>" > library/rocksdbjni/VERSION
```
2. Assemble the artifact:
```
bazel build //library/rocksdbjni:assemble-maven-mac
```

3. Deploy the artifact:
```
export DEPLOY_MAVEN_USERNAME=...
export DEPLOY_MAVEN_PASSWORD=...
bazel run //library/rocksdbjni:deploy-maven-mac -- release
```

## Assembling Manually

...

Reply:
updated!

Reply:
I think the next section should be renamed to `# Assembling Manually` as well right?

Reply:
updated again